support-data-rest/src/main/java/org/uniprot/api/support/data/configure/service/UniProtKBConfigureService.java

```diff
@@ -70,7 +70,12 @@ public List<EvidenceGroup> getGoEvidences() {
     public List<DatabaseGroup> getDatabases() {
         List<DatabaseGroup> databases =
                 Arrays.stream(UniProtDatabaseCategory.values())
-                        .filter(dbCat -> dbCat != UniProtDatabaseCategory.UNKNOWN)
+                        .filter(
+                                dbCat ->
+                                        (dbCat != UniProtDatabaseCategory.UNKNOWN
+                                                && dbCat
+                                                        != UniProtDatabaseCategory
+                                                                .GENE_ONTOLOGY_DATABASES))
                         .map(this::getDatabaseGroup)
```

Comment:
take out the filter condition to a method with meaningful name like 
boolean filteredUniProtDBCategory(dbCategory){
 // code with conditions
}